File: 1-two-sum/two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {

        int[] resposta = new int[2];

     for(int i = 0; i < nums.Length - 1; i++)
     {
        for(int j = i+1; j < nums.Length; j++){
            if(nums[i] + nums[j] == target){
            resposta[0] = i;
            resposta[1] = j;

        }
        }
     }
    
    return resposta;
    
    }
}

File: 1-two-sum/two-sum.go
func twoSum(nums []int, target int) []int {
    
    numMap := make(map[int]int)

    for i, num := range nums{
        complement := target - nums[i]

        if index, ok := numMap[complement]; ok{
            return []int{index, i}
        }
        numMap[num] = i
    }

    return nil
}